include/evmc/evmc.h

```diff
@@ -451,21 +451,35 @@ typedef void (*evmc_get_storage_fn)(struct evmc_uint256be* result,
                                     const struct evmc_address* address,
                                     const struct evmc_uint256be* key);
 
+
+/**
+ * The effect of an attempt to modify a contract storage item.
+ */
+enum evmc_storage_status
+{
+    EVMC_STORAGE_UNCHANGED = 0, /**< The storage item value unchanged. */
+    EVMC_STORAGE_MODIFIED = 1,  /**< The storage item value modified. */
+    EVMC_STORAGE_ADDED = 2,     /**< The storage item added. */
+    EVMC_STORAGE_DELETED = 3,   /**< The storage item deleted. */
```

Comment:
Does this needs to expose special cases about 0?

Adding explanation above may be good. We have the following cases, please map them to codes:
- zero -> zero (UNCHANGED? can this be MODIFIED?)
- zero -> non-zero (ADDED)
- non-zero -> zero (DELETED)
- non-zero -> non-zero (UNCHANGED/MODIFIED)

Reply:
I guess the caller can deduce that bat value == 0 && unchanged.